tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -201,7 +201,7 @@ export const handlers: Handler[] = [
                 return { resolved: false, message: 'Error parsing JSON file: ' + file };
             }
 
-            let resolved = true;
+            const resolved = true;
```

Comment:
Prob don't need this const at all, if it's just true.  Also -- looks like the prefer shorthand rule isn't working below in the return statement?